pyincore/analyses/buildingportfolio/recovery/BuildingPortfolioRecoveryAnalysis.py

```diff
@@ -230,15 +231,16 @@ def run(self):
         # Trajectory for Best Line Functionality and Full Functionality
         mean_recovery_output = sum(recovery_fp) / sample_size
 
-        output_file = output_base_name + 'building-recovery' + '.csv'
-        with open(output_file, 'w+', newline='') as output_file:
+        output_file_name = output_base_name + 'building-recovery' + '.csv'
+        with open(output_file_name, 'w+', newline='') as output_file:
             spam_writer = csv.writer(output_file, delimiter=',', quotechar='"', quoting=csv.QUOTE_MINIMAL)
             spam_writer.writerow(['Building_ID', 'Building_Lon', 'Building_Lat']
                                  + list(range(1, time_steps + 1)))
             for i in range(sample_size):
                 spam_writer.writerow([building_data['Build_ID_X'][i], building_data['X_Lon'][i],
                                       building_data['Y_Lat'][i]] + list(recovery_fp[i]))
 
+        output_file_name = output_base_name + 'portfolio-recovery' + '.csv'
```

Comment:
output_file_name is defined twice, I think we can remove the one on line 243. Also, can you maybe make this as output_base_name + "_building-recovery'? This makes it easier to find outputs for datawolf since I can make the output search for files with a wildcard like *_building-recovery.csv'

Reply:
Addressed~